study.md

```diff
@@ -25,7 +25,15 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+SPAs are Web apps that load a single HTML page and dynamically update that page as the user interacts with the app.  Instead of rendering a new HTLM page each time that the web app calls the server, resulting a browser refresh, all interaction in SPAs happens through AJAX calls.  Those calls retrun data, usually in the form of JSON, and the web app uses that data to dynamically update the page.
```

Comment:
Ideally we shoot to just make one AJAX call to the server so that we don't have to wait for server requests to dynamically update our page.